### PR TITLE
Print status message based on number of tests run

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

File: mint.sh [hunk 1]
```diff
@@ -132,7 +132,12 @@ function main()
         (( i++ ))
     done
```
Comment (balamurugana) [inline mint.sh]: You could change the above logic like `i=0` then do `(( i++ ))` before `echo -n`.  Later `i == count` for all runs and `i` has last executed test number

File: mint.sh [hunk 2]
```diff
@@ -132,7 +132,12 @@ function main()
...
 
-    echo "Finished running all tests."
+    ## Report when all tests in run_list are run
+    if [ $i -eq $((count + 1)) ]; then
+        echo -e "\nFinished running Mint. All tests run successfully"
+    else 
+        echo -e "\nFinished running Mint. Executed $((i - 1)) out of $count tests."
```
Comment (balamurugana) [inline mint.sh]: IMO `mint` saying `Mint` is not clean.  User knows he is running mint.  I think below message is good enough `$i/$count tests ran successfully` where `(( i-- ))` if `$i -ne $count`

**Comment by nitisht** [comment]
Updated @balamurugana

**Comment by poornas** [comment]
needs rebase.

**Pull request merged**